resources/copy-package-json.js

```diff
@@ -11,8 +11,20 @@
  */
 
 const fs = require('fs');
+const assert = require('assert');
 
 const package = require('../package.json');
 delete package.scripts;
 delete package.devDependencies;
+
+const { version } = package;
+const match = /^[0-9]+\.[0-9]+\.[0-9]+-?([^.]*)/.exec(version);
+assert(match, 'Version does not match semver spec.');
+const tag = match[1];
+if (tag) {
+  assert(tag !== 'rc', 'Only "rc" tag is supported.');
```

Comment:
This seems backwards, if only the 'rc' tag is supported.

Reply:
`latest` is added by default:
https://docs.npmjs.com/cli/publish#description
> By default, npm publish updates and npm install installs the latest tag

I will add a comment to make it more clear.

Reply:
@mjmahone I change the code to explicitly set `latest`.
Can you please review changes.